src/main/resources/data.sql

```diff
@@ -1,118 +1,19 @@
-/* SPORTS */
 INSERT INTO ACTIVITY_PRESET (ID, ACTIVITY_NAME, ACTIVITY_CATEGORY, GOOGLEPOICATEGORY, GOOGLEPOIKEYWORD, IMAGEURL)
 VALUES (1, 'Play Football', 'SPORTS', '???', '???', '???'),
-       (2, 'Hiking on the Uetliberg', 'SPORTS', '???', '???', '???'),
```

Comment:
Again: Why are ActivityPresets deleted?

Reply:
They are not deleted, the reason why they are not here is because this pr is based on the older version of master where we did not have any activityPresets.